opre_ops/ops_site/migrations/0007_auto_20210923_1822.py

```diff
@@ -36,7 +36,7 @@ class Migration(migrations.Migration):
             name='CANAmount',
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('fiscal_year', models.IntegerField(max_length=4)),
+                ('fiscal_year', models.DecimalField(decimal_places=0, max_digits=4)),
```

Comment:
[suggestion][blocking]: The [resource here](https://newbedev.com/year-field-in-django) suggests the use of validating and bounding the year.  We should consider using some form of that, so we can prevent numbers that are not year to be rejected before saving the record.